hack/dockerfile/binaries-commits

```diff
@@ -11,5 +11,5 @@ VNDR_COMMIT=c56e082291115e369f77601f9c071dd0b87c7120
 BINDATA_COMMIT=a0ff2567cfb70903282db057e799fd826784d41d
 
 # CLI
-DOCKERCLI_REPO=https://github.com/docker/cli
-DOCKERCLI_COMMIT=c3648a9c9400d45524cc71b8fca4085b192c626f
+DOCKERCLI_REPO=https://github.com/aaronlehmann/cli
+DOCKERCLI_COMMIT=fd5a5910409fe5ed862b29de45a66f2bf8056894
```

Comment:
We can move the tests to a separate PR.

Opts package should really not be in this repo.
We're only using a couple of functions from it for cmd/dockerd.